src/EditorFeatures/CSharpTest/NavigateTo/NavigateToSearcherTests.cs

```diff
@@ -33,66 +33,78 @@ public class NavigateToSearcherTests
             if (isFullyLoaded)
             {
                 // First do a full search
-                searchService.Setup(ss => ss.SearchProjectAsync(
-                    It.IsAny<Project>(),
+                searchService.Setup(ss => ss.SearchProjectsAsync(
```

Comment:
note: this MOQ goo is for some very low level tests that actually confirm we call the service in particular ways depending on if hte solution is fully loaded or not.  it's very ugly and not critical to understand :D

Reply:
Can we instead hand-write the implementation of the interface? I find that much better than Moq. See e.g. `MockEditAndContinueWorkspaceService.cs`